app/models/Organization.scala

```diff
@@ -123,6 +143,21 @@ object BankaccountOrganization{
   implicit val bankaccountOrganizationJsonFormat = Json.format[BankaccountOrganization]
 }
 
+trait BankaccountOrganizationNameBase{
```

Comment:
`BankAccount` instead of `Bankaccount` ?

Reply:
For models we write the first letter as capital letters. For relations like ProfileOrganization we can easily find the related models. So i think we better use Bankaccount because its a model not a relation

Reply:
No, it's about readability of code. We have to use UpperCamelCase as it is a naming convention from the very beginning (https://en.wikipedia.org/wiki/Naming_convention_(programming)#Java). ^^
Question is not, **if** we should use it, but if we can change it easily now?